orders/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from warehouses.models import Item
+
+
+class Order(models.Model):
+    """
+    Ordered items made by User.
+    """
+    item_id = models.ForeignKey(Item, on_delete=models.CASCADE)
```

Comment:
Confirm if we really need `CASCADE` here. Remember that when we cascade DB tables, it means that when data is hard deleted, all other related data will also be deleted. Is that what we want?

Reply:
Ok. I got your point.
Addressed here: https://github.com/jino-d/warehouse/pull/1/commits/59ace3fed4a512764e82158d1ee809b3ee45eeb7